runner/runs/quad_multi_mixed_vel_col_func.py

```diff
@@ -0,0 +1,27 @@
+from runner.run_description import RunDescription, Experiment, ParamGrid
+
+_params = ParamGrid([
+    ('seed', [0000, 3333]),
+    ('quads_collision_vel_penalty_mode', ['none', 'linear', 'quadratic']),
+    ('quads_collision_vel_penalty_radius', [4.0, 6.0]),
+    ('quads_collision_smooth_vel_coeff', [0.5, 2]),
+])
+
+_experiment = Experiment(
+    'grid_search_vel_collision_func-8_mixed',
+    'python -m run_algorithm --env=quadrotor_multi --train_for_env_steps=5000000000 --algo=APPO --use_rnn=False --num_workers=36 --num_envs_per_worker=4 --learning_rate=0.0001 --ppo_clip_value=5.0 --recurrence=1 --nonlinearity=tanh --actor_critic_share_weights=False --policy_initialization=xavier_uniform --adaptive_stddev=False --hidden_size=256 --quads_neighbor_hidden_size=256 --quads_neighbor_encoder_type=attention --neighbor_obs_type=pos_vel --with_vtrace=False --max_policy_lag=100000000 --gae_lambda=1.00 --max_grad_norm=5.0 --exploration_loss_coeff=0.0 --rollout=128 --batch_size=1024 --quads_use_numba=True --quads_num_agents=8 --quads_episode_duration=15.0 --quads_mode=mix --quads_formation_size=0.0 --encoder_custom=quad_multi_encoder_deepset --with_pbt=False --quads_collision_reward=5.0 --quads_collision_hitbox_radius=2.0 --quads_collision_falloff_radius=4.0 --quads_collision_smooth_max_penalty=10.0',
```

Comment:
Any reason these use the deepsets encoder? Attention wasn't merged yet?

Reply:
So, it's not the encoder_custom=quad_multi_encoder_deepset that decides which neighbor encoder that we use.

Reply:
We use quads_neighbor_encoder_type to decide. And I have already set quads_neighbor_encoder_type=attention here.

Reply:
Oh yeah. We should definitely fix this.

Reply:
How about merge it first? And we can discuss it tomorrow. And I can take charge of it, if you want.

Reply:
Which PR? This one I merged.

Reply:
I mean the current PR, since you have already merged, just ignore my reply.

Reply:
You should go rest :) 
I recognise this is a lot to work!! Sleep well and see you tomorrow :)